httpstan/models.py

```diff
@@ -278,14 +280,17 @@ def _redirect_stderr_to(stream: IO[Any]) -> int:
         # Note: `library_dirs` is only relevant for linking. It does not tell an extension
         # where to find shared libraries during execution. There are two ways for an
         # extension module to find shared libraries: LD_LIBRARY_PATH and rpath.
+        libraries = ["protobuf-lite", "sundials_cvodes", "sundials_idas", "sundials_nvecserial", "tbb"]
+        if platform.system() == "Darwin":
+            libraries.extend(["tbbmalloc", "tbbmalloc_proxy"])
```

Comment:
Why these are needed on macOS

Reply:
I don't know. Stan Math builds them for macOS and not for Linux. @wds15 do you want to comment?

Reply:
The tbbmalloc things are not needed on macOS, but they speed up Stan by 10-15%...even for single thread performance. Other platforms do not benefit that much or not at all.